### PR TITLE
Cypher Editor - define list of commands with subcommands

### PR DESCRIPTION
**Pull request by FylmTM**
PR should provide extensive list of all commands and their subcommands support by browser.

Picture:
image

### CODE REVIEW

File: src/browser/modules/Editor/language/consoleCommands.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  },
+  {
+    name: ':clear',
+    description: ' - remove all frames from the stream'
+  },
+  {
+    name: ':config'
+  },
+  {
+    name: ':schema - display indexes and constraints'
```
Comment (apcj) [inline src/browser/modules/Editor/language/consoleCommands.js]: ` - display indexes and constraints` should be part of the description, not part of the name

File: src/browser/modules/Editor/language/consoleCommands.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      { name: 'load csv' },
+      { name: 'unwind' },
+      { name: 'start' },
+      { name: 'create-unique' },
+      { name: 'create-index-on' },
+      { name: 'starts-with' },
+      { name: 'ends-with' },
+      { name: 'contains' },
+      { name: 'rest' },
+      { name: 'rest-get'},
```
Comment (apcj) [inline src/browser/modules/Editor/language/consoleCommands.js]: Lines 35-38 are breaking the linter; they need a space before the `}`

File: src/browser/modules/Editor/language/consoleCommands.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      { name: 'rest-put'},
+      { name: 'rest-delete'},
+      { name: 'rest-post'}
+    ]
+  },
+  {
+    name: ':play',
+    description: ' - loads a mini-deck with either guide material or sample data',
+    commands: [
+      { name: 'intro' },
```
Comment (apcj) [inline src/browser/modules/Editor/language/consoleCommands.js]: There are loads more guides than this list, but I'm sure we can the missing ones in later

Comment (FylmTM) [inline src/browser/modules/Editor/language/consoleCommands.js]: @apcj I have checked all other guides in the repo.
Looks like that I added all guides that are working right now and make sense to add them.

Other ones are either not working. Or are duplicates (there 2 guides for movie graphs, 3 guides for graph concenpts).

**Comment by FylmTM** [comment]
@apcj 
TL;DR: - I have updated code to fix linting issues and also added missing help & guides. And added some descriptions too.

----

The whole system of how commands are handled, and help topics/guides are loaded is needed to be reviewed. It is hard to pull some information out.

Also, a bunch of guides & help topics is outdated/not converted. And information in guides itself should be re-checked.

For example in:
* In `:play start` if you click on 'start learning' it will trigger `:play concepts`
* In `:help play` you can see that there is `:play graphs`

This is 2 different guides. But the content is more-or-less same (differs in details).


Ideally all commands should be defined somewhere, and be used both for execution and for editor schema definition.

**Comment by oskarhane** [comment]
Can I merge this now @FylmTM?

**Comment by apcj** [comment]
@oskarhane I think the DO NOT MERGE applied to a new release of cypher-editor, which happened and this now depends on.

**Comment by oskarhane** [comment]
@apcj I think so too, just wanted to be sure. 😸

**Comment by FylmTM** [comment]
@oskarhane 

image
